pom.xml

```diff
@@ -198,6 +199,11 @@
         <artifactId>picocli</artifactId>
         <version>${dep.picocli.version}</version>
       </dependency>
+      <dependency>
```

Comment:
This dependency also needs to be included in the `dependencies` section of the pom.